### PR TITLE
Prison Cryo

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request

Adds a cryo to the prison

## How This Contributes To The Skyrat Roleplay Experience

Dragging prisoners out of the prison to the pubbie cryo room is dumb

## Proof of Testing

image

## Changelog

:cl:
qol: Prisons now have a cryo unit again
/:cl:

### CODE REVIEW

**Comment by gamermomentum** [comment]
Is there no better place for the cryo pod than *dead center* on the little basketball halfcourt?

**Comment by YakumoChen** [comment]
its random

**Comment by Jolly-66** [comment]
Not a fan of it being 100% random.
The logic can be improved to make it stick to a wall.
You're also missing the adjacent computer console too.

File: modular_skyrat/modules/automapper/code/area_spawn_entries.dm [hunk 1]
```diff
@@ -50,7 +50,12 @@
 	desired_atom = /obj/machinery/vending/dorms
 	mode = AREA_SPAWN_MODE_HUG_WALL
 
+/datum/area_spawn/prison_cryo
+	target_areas = list(/area/station/security/prison, /area/station/security/prison/shower)
+	desired_atom = /obj/machinery/cryopod
```
Comment (Zonespace27) [inline modular_skyrat/modules/automapper/code/area_spawn_entries.dm:55]: exclude void raptor from this

**Comment by YakumoChen** [comment]
> Not a fan of it being 100% random. The logic can be improved to make it stick to a wall. You're also missing the adjacent computer console too.

Tried the console and couldn't get it to spawn in the prison on a wall mount with several tests.

**Comment by Tastyfish** [comment]
To be honest most of the prison designs just don't have any room to even intentionally place all the cryo gear without further remapping. I have a vague idea how to fix this though: making the beds double as cryo pods in some way.

**Comment by YakumoChen** [comment]
That would still require remapping, and this is the least intrusive thing I can think of to add it.

**Comment by Zonespace27** [comment]
Cleaning up old PRs unlikely to merge in their current state (107 is too many dear god)

**Pull request closed without merging**